content/news/2022-09-02-gsoc-final-blog-david-chocholaty.md

```diff
@@ -0,0 +1,632 @@
+title: GSoC 2022 Work Product - Pitch Shift effect and Group delay handling
+authors: David Chocholatý
+status: draft
+tags: TODO(davidchocholaty) add tags
+comments: yes
+
+#### Introduction
+The project implements the Pitch Shift effect for the Mixxx DJ
+application. The Pitch Shift effect raises or lowers the original pitch
+of an audio signal. Thanks to the long working period, the project was expanded
+with the implementation of the Group Delay handling for the effect chain.
+
+#### Motivation
+Before the effect is implemented, the pitch could be changed using the deck's
+rate slider separately only. This imposes significant restrictions on the ways
+other effects can interact with the sound. In this project, the new Pitch Shift
+effect is introduced in the built-in effects, which can be used
+in the effect chain. It implements the wish
+[Add a Transpose / Pitch Shift effect](https://github.com/mixxxdj/mixxx/issues/7389)
+for the Mixxx application. The effect has to work with the effect chain API.
+With that, the other extensional options can be used. Primarily, it allows
+for the user to use a much wider range, than the Pitch Shifter
+for the deck player. Due to the produced latency based on the pitch processing,
+the production delay has to be handled for the wet/dry or wet+dry mode
+to ensure that the original (dry) and processed (wet) signals overlap.
+
+#### Background
+List of terms from the field of music, sound processing
+and development in general:
+
+* Scale
+  * In music theory, a scale is any set of musical notes ordered
```

Comment:
This indentation is lost in the rendered html

Reply:
Thank you for the hint. Now, it should be fixed.